plugin-api/Examples/react-plugin/app.jsx

```diff
@@ -4,22 +4,22 @@ import { Metaflow } from '../../MetaflowPluginAPI'
 import './style.css';
 
 function App() {
-  const [run, setRun] = useState(null)
+  const [metadata, setMetadata] = useState(null)
 
-  // subscribe to listen run resource.
+  // subscribe to listen metadata resource.
   useEffect(() => {
-    Metaflow.subscribe(['run'], (event) => {
-      setRun(event.data);
+    Metaflow.subscribeToMetadata((event) => {
+      setMetadata(event.data);
     });
-    Metaflow.setHeight(100)
```

Comment:
trying to understand here why setHeight(100) was replaced by setHeight()?

Reply:
setHeight has optional fixedHeight parameter. When no parameter is given, API will try to find out the full height of document and use that. There wasn't really need for fixed height here.